dask/dataframe/io/hdf.py

```diff
@@ -8,14 +8,21 @@
 from fsspec.utils import build_name_function, stringify_path
 from tlz import merge
 
-from dask import config, multiprocessing
+from dask import config
 from dask.base import compute_as_if_collection, get_scheduler, tokenize
 from dask.dataframe.core import DataFrame
 from dask.dataframe.io.io import _link, from_map
 from dask.dataframe.io.utils import DataFrameIOFunction
 from dask.delayed import Delayed, delayed
 from dask.utils import get_scheduler_lock
 
+try:
+    from dask import multiprocessing
+except ImportError:
+    mpget = object()
+else:
+    mpget = multiprocessing.get
```

Comment:
Can probably simplify this too. Maybe something like so?

```suggestion
MP_GET  = named_schedulers.get("processes", object())
```

Also capitalizing since this was done elsewhere. Though no strong feelings on that